spec/models/article_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+
+  let(:article) { create(:article) }
+
+  it 'is valid with all valid attributes' do
+    expect(article).to be_valid
+  end
+
+  it 'is not valid without a title' do
+    article.title = nil
+    expect(article).to_not be_valid
+  end
+
+  it 'is not valid without content' do
+    article.content = nil
+    expect(article).to_not be_valid
+  end
+
+  it 'is not valid without an user' do
+    article.user = nil
+    expect(article).to_not be_valid
+  end
+end
```

Comment:
I don't quite agree with this. While I do agree you shouldn't test built in behaviour, `Article` is a custom class and she's testing that fields that have been determined to be required are actually required. This is something that you do have to specify on attributes in classes that extend `ActiveRecord::Base`. So therefore, I'd say it's a valid test, because it's something that you could very easily forget to do, or maybe accidentally update at some stage.

Reply:
I see your point, and it's valid. Although I'd prefer this kind of test being associated with the Controller, but since Rails provides this validation at Model level, I'm fine with this.

Reply:
I found that these tests were the most useful when doing TDD, it's interesting that you have such an opposing opinion! 

I'm of the opinion that these tests serve a valuable purpose and would like to continue making them. I see there are many examples within Portal that match my tests also.